Java/policy-service/src/main/java/com/insuranceapplication/policyservice/methods/Utils.java

```diff
@@ -1,23 +1,146 @@
+    private InsuredObject insuredVehicle;
+    private InsuredObject insuredDriver;
+    private Customer customer;
+    private List<PremiumCalcConfigValue> calcVariables;
+    private List<ObjectRiskConfig> objectRiskConfigs;
+    private RestTemplate template = new RestTemplate();
+    private ObjectRisk riskToUpdate = new ObjectRisk();
+
+    public static Object mapToObject(LinkedHashMap map, Class clazz) {
         ObjectMapper mapper = new ObjectMapper();
-        Object o = mapper.convertValue(map,clazz);
+        Object o = mapper.convertValue(map, clazz);
         return o;
     }
-    public static List mapToList(List<LinkedHashMap> mapList,Class clazz){
+    public static List mapToList(List<LinkedHashMap> mapList, Class clazz) {
 
         List<Object> list = new ArrayList<>();
-        for (LinkedHashMap map : mapList){
-            list.add(mapToObject(map,clazz));
+        for (LinkedHashMap map : mapList) {
+            list.add(mapToObject(map, clazz));
         }
         return list;
     }
 
+    public static Double percentToDouble(String percentage) {
+        return Double.parseDouble(percentage.replace(",", ".").replace("%", "")) / 100;
+
+    }
+
+    public Integer yearsFromNow(Date date) {
+        if (date == null) {
+            date = new Date();
+        }
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd");
+        return Math.abs(Period.between(LocalDate.parse(sdf.format(date)), LocalDate.now()).getYears());
+    }
+
+    public Double getDoubleFromPrecentage(String precentage, Double value) {
+        Double precentageOfBonus = Double.valueOf(precentage.replace(",", ".").replace("%", ""));
+        Double sumToAdd = value.floatValue() * (precentageOfBonus / 100.0D);
+        DecimalFormat df = new DecimalFormat("###.##", new DecimalFormatSymbols(Locale.US));
+        return Double.valueOf(df.format(sumToAdd));
+
+    }
+
+    public Customer getCustomerFromInsuredObject(InsuredObject driver) {
+        Customer custDriver = new Customer();
+        custDriver.setId(driver.getN01());
+        ResponseEntity response = template.postForEntity(eurekaClient.getApplication(Variables.dbName)
+                .getInstances().get(0).getHomePageUrl() + "/searchcustomers", custDriver, List.class);
+        ArrayList listFromResponse = (ArrayList) Utils.mapToList((List<LinkedHashMap>) response.getBody(), Customer.class);
+        return (Customer) listFromResponse.get(0);
+    }
+
+    public Vehicle getVehicleFromInsuredObject(InsuredObject insuredVehicle) {
+        Vehicle vehicle = new Vehicle();
+        vehicle.setId(insuredVehicle.getN01());
+        ResponseEntity response = template.postForEntity(eurekaClient.getApplication(Variables.dbName)
+                .getInstances().get(0).getHomePageUrl() + "/getvehicle", vehicle, Vehicle.class);
+        return (Vehicle) response.getBody();
+    }
+
+    public InsuredObject findInsuredObject(Policy policy, String objectType) {
```

Comment:
Takie metody nie powinny znajdować się w klasie przechowującej metody ogólne.  To powinno zawierać się w miejscu w którym to będzie wykorzystane ew możnaby ją umieścić w service.